### DISCUSSION TITLE
Added Wavelog to Proxmox VE Helper-Scripts

### DISCUSSION THREAD

**Discussion by DonLocke** [wavelog/wavelog#1211] [Show and tell]
I created a script in the [Proxmox VE Helper-Scripts](https://github.com/community-scripts/ProxmoxVE) repository for easy installation on Proxmox via LXC.

Check it out here: https://community-scripts.github.io/ProxmoxVE/scripts?id=wavelog

**Comment by HB9HIL** [comment]
Thats nice :)

**Reply by HB9HIL** [reply]
PS: We can't officially announce this script as it does not match our installation steps in the Wiki. But thanks for putting effort in it and probably it will help others 😃

**Reply by HB9HIL** [reply]
Besides thats.. Works nice. Just tested in my personal pve. Good job

**Reply by DonLocke** [reply]
Hey I'm glad you like it! I tried to adapt the install process from the wiki for Linux servers, is `php-gd` a dependency that's being added for a future version of wavelog? I didn't see it listed as a prerequisite on the wiki.

**Reply by HB9HIL** [reply]
https://github.com/community-scripts/ProxmoxVE/pull/292

**Reply by HB9HIL** [reply]
>  I didn't see it listed as a prerequisite on the wiki.

I adjusted the wiki. Now it's there. You couldn't know 🙂
